ProxyAppsSuite/tests/ember.yaml

```diff
@@ -0,0 +1,470 @@
+                regex: '\ + \d+\ +\S+\ +\S+\ +(\S+)\ +\S+\ +\S+'
+                action: store
+                files: pingpong.out
+            kMsg_per_S:
+                regex: '\ + \d+\ +\S+\ +\S+\ +\S+\ +(\S+)\ +\S+'
+                action: store
+                files: pingpong.out
+            mb_per_s:
+                regex: '\ + \d+\ +\S+\ +\S+\ +\S+\ +\S+\ +(\S+)'
+                action: store
+                files: pingpong.out
+
+
+
+
+sweep3d:
+    inherits_from: _no_build
+    summary: Communication sweeping
+    run:
+        cmds:
+            - cd mpi/sweep3d
+            - ls
+            - srun -n 8 ./sweep3d -pex 4 -pey 2 -iterations 1000 -nx 300 -ny 300 -nz 350 -sleep 1000 -vars 20 -kba 25 &> sweep3d.out
+            - cat sweep3d.out
+            - mv sweep3d.out ../../sweep3d.out
+    result_parse:
+        regex:
+            passed:
+                regex: '#\sPx\:\s+(\d+)*'
+                action: 'store_true'
+                files: sweep3d.out
+            px:
+                regex: '\#\sPx\:\s+(\d+)*'
+                action: store
+                files: sweep3d.out
+            py:
+                regex: '\#\sPy\:\s+(\d+)*'
+                action: store
+                files: sweep3d.out
+            nx:
+                regex: '\#\sNx\sx\sNy\sx\sNz\:\s+(\d+)'
+                action: store
+                files: sweep3d.out
+            ny:
+                regex: '\#\sNx\sx\sNy\sx\sNz\:\s+\d+\sx\s+(\d+)\sx\s+\d+'
+                action: store
+                files: sweep3d.out
+            nz:
+                regex: '\#\sNx\sx\sNy\sx\sNz\:\s+\d+\sx\s+\d+\sx\s+(\d+)'
+                action: store
+                files: sweep3d.out
+            kba:
+                regex: '\#\sKBA\:\s+(\d+)'
+                action: store
+                files: sweep3d.out
+            variables:
+                regex: '\#\sVariables\:\s+(\d+)'
+                action: store
+                files: sweep3d.out
+            iterations:
+                regex: '\#\sIterations\:\s+(\d+)'
+                action: store
+                files: sweep3d.out
+            time:
+                regex: '\s+ (\S+)\s+ \S+\s+ \S+'
+                action: store
+                files: sweep3d.out
+            kbytes_exchange_per_rank_max:
+                regex: '\s+ \S+\s+ (\S+)\s+ \S+'
+                action: store
+                files: sweep3d.out
+            mb_per_s_per_rank:
+                regex: '\s+ \S+\s+ \S+\s+ (\S+)'
+                action: store
+                files: sweep3d.out
+
+
+hotspotinc:
+    inherits_from: _build
+    summary: Hotspot random network access
+    run:
+        cmds:
+            - export PATH=$(find . -name "shmemrun" -exec dirname $(readlink -f '{}') \;):${PATH}
+            - export LD_LIBRARY_PATH=$(find . -name "liboshmem.so" -exec dirname $(readlink -f '{}') \;):${LD_LIBRARY_PATH}
+            - 'cd shmem/hotspotinc || (echo "$0 at $LINENO FAILED" ; exit 1 ;)'
```

Comment:
I'm still trying to refine this, as I found that when the `cmd || ( cmd; exit 1; )` doesn't really do what we want it to do.  I recommend removing it until we can figure out a solid plan for that.